app/locales/es-AR.json

```diff
@@ -148,7 +148,11 @@
   "If you use this computer with multiple users, please use this link to update your status.": "Si utiliza este equipo con varios usuarios, por favor utilice este enlace para actualizar su estado.",
   "Write it down somewhere or save it on your computer.": "Anótela en algún lugar o guardelo en su computador.",
   "The link has been copied.": "El enlace se ha copiado.",
+<<<<<<< HEAD
```

Comment:
This needs to get removed :) It's basically git way of saying merge conflict. Just delete the line :)